### PR TITLE
VOTE-2105: Temporary CSS adjustment to in-page nav

### PR DESCRIPTION
**Pull request by rkoch2**
## Jira ticket

[VOTE-2105](https://cm-jira.usa.gov/browse/VOTE-2105)

## Description

When clicking the name of a section on the USWDS in-page nav component, the "current section" indicator does not move to the correct section. This temporary adjustment hides the current section indicator by removing the text color adjustment and left indicator bar, until the bug can be resolved with USWDS (issue [here](https://github.com/uswds/uswds/issues/5953)).

Before (darker color and left indicator show which section is currently on-screen):
image

After change (no visual indication of current section):
image

### Post-deploy steps

Navigate to `web/themes/custom/votegov`, and run `npm run dev`
In the root directory, run `lando retune`

### QA/Testing instructions

1. Navigate to a page where the in-page nav appears -- for example, Guide to Voting > Voting as a college student
2. Make sure that the in-page nav appears on the right side of the page
3. Scroll through the different sections of the page, making sure that there are no visual changes to the in-page nav (ie. changes that would indicate which section of the page is currently on-screen)
4. Interact with the in-page nav by clicking a section name to ensure that it still works as expected (ie. scrolls so that the given section is on-screen)

## Checklist for the Developer

- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [x] The file changes are relevant to the task objective.
- [x] Code is readable and includes appropriate commenting.
- [x] Code standards and best practices are followed.
- [x] QA/Test steps were successfully completed, if applicable.
- [x] Applicable logs are free of errors.

### CODE REVIEW

File: web/themes/custom/votegov/src/sass/uswds-overrides/usa-in-page-nav.scss [hunk 1]
```diff
@@ -22,12 +22,15 @@
     }
 
     &.usa-current {
-      @include vote-link-bg($color: $base-dark);
+      // @include vote-link-bg($color: $base-dark);
+      @include vote-link-bg; // temporarily using default color
 
+      // section indicator (left of current section heading)
```
Comment (clmedders) [inline web/themes/custom/votegov/src/sass/uswds-overrides/usa-in-page-nav.scss:28]: i dont think this comment is needed... i would change to something like. "Turning off style untill USWDS fix is made"

File: web/themes/custom/votegov/src/sass/uswds-overrides/usa-in-page-nav.scss [hunk 2]
```diff
@@ -22,12 +22,15 @@
     }
 
     &.usa-current {
-      @include vote-link-bg($color: $base-dark);
+      // @include vote-link-bg($color: $base-dark);
```
Comment (clmedders) [inline web/themes/custom/votegov/src/sass/uswds-overrides/usa-in-page-nav.scss:25]: i would take this line out... just have it as `@include vote-link-bg` and then make a comment above that after the fix to add back `($color: $base-dark)`

File: web/themes/custom/votegov/src/sass/uswds-overrides/usa-in-page-nav.scss [hunk 3]
```diff
@@ -22,12 +22,15 @@
...
+      @include vote-link-bg; // temporarily using default color
 
+      // section indicator (left of current section heading)
       &:after {
         @include u-left(0);
         inset-inline-start: 0;
         background-color: $base-dark;
+        visibility: hidden; // temporarily hiding section indicator
       }
```
Comment (clmedders) [inline web/themes/custom/votegov/src/sass/uswds-overrides/usa-in-page-nav.scss:34]: i would use `display: none;` and add it to top of the list, can even comment out the rest to show that it is supposed to be turned off.

**Review by clmedders — CHANGES_REQUESTED** [review]
see above comments

**Review by clmedders — APPROVED** [review]

**Review by rayestrada — APPROVED** [review]

**Pull request merged**